vendor/diff-parsers-ext-parsewyc.patch

```diff
@@ -1,5 +1,47 @@
 --- ocaml-4.13-extended/lexer.mll
 +++ parse-wyc/lib/lexer.mll
+@@@@
+ let is_keyword name = Hashtbl.mem keyword_table name
+ 
+ let check_label_name lexbuf name =
+   if is_keyword name then error lexbuf (Keyword_as_label name)
+ 
+-(* To "unlex" a few characters *)
```

Comment:
Don't we need the same fix in parse-wyc ?

Reply:
This is only necessary to print the disambiguated identifiers, that should be called by ocaml-lsp, which wouldn't set the `--numeric` option (only set to get the re-indent behavior in emacs) that is the only way to run parse-wyc for recovery parsing.